packages/nx-go/src/utils/go-bridge.ts

```diff
@@ -34,6 +34,32 @@ export const getGoShortVersion = (): string => {
   return `${major}.${minor}`;
 };
 
+/**
+ * Executes the `go list -m -json` command in the
+ * specified directory and returns the output as a string.
+ *
+ * @param cwd the current working directory where the command should be executed.
+ * @param failSilently if true, the function will return an empty string instead of throwing an error when the command fails.
+ * @returns The output of the `go list -m -json` command as a string.
+ * @throws Will throw an error if the command fails and `failSilently` is false.
+ */
+export const getGoModules = (cwd: string, failSilently: boolean): string => {
+  try {
+    return execSync('go list -m -json', {
+      encoding: 'utf-8',
+      cwd,
+      stdio: ['ignore'],
+      windowsHide: true,
```

Comment:
I didn't know about this option, but it avoids having a terminal that appears and then disappears on Windows. I'd noticed it already, it will be fixed 💯